docker-compose.override.yaml

```diff
@@ -13,3 +11,10 @@ services:
     build: ./app
     volumes:
       - ./app:/home/python/app
+
+  nginx:
+    volumes:
+      - ./nginx/certs:/certs
+    ports:
+      - "80:80"
+      - "443:443"
```

Comment:
Should we really use these ports for local development? Why not e.g. 5000 like we had before?

Reply:
Laziness. Why type `localhost:5000` if you can type `localhost` :)